docs/developer/development/ci.rst

```diff
@@ -0,0 +1,146 @@
+Continuous Testing
+==================
+
+Add a new test in the continuous integration system
+---------------------------------------------------
+
+A test as continuous integration system level is an End-to-End test that need a
+dedicated environment (with one or several machines (virtual or container)).
```

Comment:
Not sure for this one:
```suggestion
When we refer to test, at continuous integration system level, it means an end-to-end test
that requires a dedicated environment, with one or several machines (virtual or container).
```

Reply:
If i could propose then =>
`At a continuous integration system level, a test is an end-to-end (operation or task) that requires a dedicated environment with one or several machines(virtual or container)`

Reply:
Done, a mix of both (I think `task` is good and also cover @gdemonet comment https://github.com/scality/metalk8s/pull/2382#discussion_r406150647